app/assets/stylesheets/sections/milestone.scss

```diff
@@ -1,3 +1,4 @@
-.issues-sortable-list .str-truncated {
-  max-width: 70%;
+.issues-sortable-list .avatar {
+  margin-right: 0px;
```

Comment:
Properties should be ordered margin-left, margin-right<br>`0px` should be written without units as `0`